### ISSUE TITLE
Feature: Feed critical failures into system logging

### ISSUE THREAD

**Issue by cpaelzer** [CanonicalLtd/ubuntu-advantage-client#2999]
TL;DR: how would a shop using central logging and monitoring realize things went wrong on a node.

I know we trace general pro things in /var/log/ubuntu-advantage*, and for 99% this is totally fine.

But I can think of the following situation that could benefit from an extra signal:
1. One is attached to pro
2. They have not one but many machines
3. They use central log management and error detection

Now in this case they could try to leverage /var/log/ubuntu-advantage* in their custom way.
But critical issues that would prevent access to security fixes IMHO might benefit from a bit of extra signal.

But what I'd suggest is that this category "you think you have Pro, but it can't work for some reason".
I'd assume apt auto-updates would complain to the system log and journal if they fail - could we check that?

The solution I'm looking for is to emit a warning to the syslog/journal in those cases.
Many already remote collect those and monitor them for critical issues throughout their server farms.

Disclaimer: I might not even know all we already have, as I do remember we worked on e.g. expiration messaging.

But even if we have it all - it might still be a great document "How to monitor Ubuntu Pro across many systems".
Over time as we add all the API and machine readable things that doc could link there in case anyone wants to go deeper (which then is close to the ideas for cloud-dashboards and such). Maybe a grafana exporter, ... I get out of control, for today this is only about "classic" central logging.

**Comment by orndorffgrant** [comment]
There is apprehension in the pro team on whether this is really desirable or not. APT should be reporting any errors from e.g. failed authentication from an expired contract. We could potentially insert a syslog warning if pro is expired, but that would not add much information and is a contractual message rather than a technical message. WDYT @cpaelzer?

**Comment by lucasmoura** [comment]
@cpaelzer just double checking if you think this issue is still relevant given @orndorffgrant comment
